### ISSUE TITLE
The package 'events-ics' was not found in the Grav repository.

### ISSUE THREAD

**Issue by tiktuk** [aoloe/grav-plugin-events-ics#1]
It is not possible to install the plugin with GPM even though it says so in the readme.

**Comment by aoloe** [comment]
yep, the README is the boilerplate create by the grav's own plugin generator.

since i'm probably the only one using this plugin, i have a hard time to know if it is worth to be added to the plugins lists...

as soon as somebody has tested it and and we are two using it, i will be glad to check how to get it added to the offical list!

**Comment by tiktuk** [comment]
Yeah, I tried today but couldn't get it to work actually.

**Comment by aoloe** [comment]
i still do not have in production, but i gave it a try on the local server and it worked.

but, as the README says it's still tightly coupled to the first site where i plan to be using it.  
i'll try to spend a little time to make it more general and then report in here what's the status.

**Comment by aoloe** [comment]
Ok, both plugins have hard coded strings that are specific to the project.

Right now, I don't have the time to make them more general, but I will open tickets that hint to what has to be modified to make them usable for other projects / other people.

I will also adapt the README file to the current status of the plugin.

**Comment by tiktuk** [comment]
Thanks. If I have time I'll also see if I can make it work once more. I need some way to generate an iCal feed.
